strapi/codemods
# add unit tests

## What does it do 

Adds unit tests for #23 

## How to test it

Confirm the unit tests provide adequate coverage for the PR mentioned above.

That PR was tested as follows:
- Clone this v3 repo that has all the relations for a content-type https://github.com/markkaylor/strapi-v3-test-migration
- Clone this v4 repo https://github.com/markkaylor/strapi-v4-test-migration
- On the v3 app run the codemods scripts `../path/to/codemods/bin/cli.js migrate:application <v3-application-path>` 
- Compare the relations to the v4 repo's relations 
- Confirm all files and folders were moved correctly based on https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/file-structure.html
